web/src/routes/(user)/people/+page.svelte

```diff
@@ -209,6 +293,22 @@
   };
 </script>
 
+{#if showMergeModal}
+  <FullScreenModal on:clickOutside={() => (showMergeModal = false)}>
+    <SuggestMerge
+      bind:personMerge1
+      bind:personMerge2
+      {potentialMergePeople}
+      on:close={() => (showMergeModal = false)}
+      on:differentFaces={() => {
```

Comment:
Since this has more than one line, let's extract the method's body to the local function